rct229/rulesets/ashrae9012019/section22/section22rule17.py

```diff
@@ -40,6 +41,10 @@ def create_data(self, context, data):
 
         return {"heat_rejection_loop_ids_b": heat_rejection_loop_ids_b}
 
+    def is_applicable(self, context, data=None):
```

Comment:
`is_applicable` is called before the `create_data`
Move this function before the `create_data`

The same applies to the other two rules.

Reply:
All set